mtags/src/main/scala-2/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -73,34 +102,28 @@ trait AutoImports { this: MetalsGlobal =>
               }
               .takeWhile(_.isInstanceOf[Import])
               .lastOption
-            val lastImportLine = lastImportOpt
+
+            val offset = lastImportOpt
               .map(_.pos.focusEnd.line)
-              .getOrElse(0) // if no previous import, add the new one at the top
+              .map(pos.source.lineToOffset)
+              .getOrElse(
+                firstBodyLineOffset(obj.impl.pos)
```

Comment:
It might be easier to get the position of  `/*<script>*/` that represents the start of `sc` content 🤔

Reply:
Oops, that doesn't work because as you mentioned, scala-cli script will have something like

```scala
/*<script>*///> using scala "2.13.8"
println(1)
```

Reply:
Maybe we can rely on `/*<script>*/` because skipping using directive is done by `offsetAfterUsingDirectives` ?

Reply:
yep - might be an option
I've checked what exactly ammonite pass as a text and it has a similar marker -  `/*<start>*/`

Reply:
Yeah it should be reliable :+1: 
https://github.com/com-lihaoyi/Ammonite/blob/bec02ae120eca3002b19405ddaea71b1fbbcaed9/amm/compiler/interface/src/main/scala/ammonite/compiler/iface/CodeWrapper.scala#L33

Reply:
reworked it - now it's based on location of `/*<script>*/`